ports/rp2/boards/VCC_GND_YD_RP2040/deploy.md

```diff
@@ -0,0 +1,8 @@
+### Flashing via UF2 bootloader
+
+To get the board in bootloader mode ready for the firmware update, execute
+`machine.bootloader()` at the MicroPython REPL.  Alternatively, hold
+down the BOOT button while pressing reset (RST).  The uf2 file below
```

Comment:
File does not includes links to precompiled UF2 binaries. Please provide the links for each version (4MB, 8MB, 16 MB)

Reply:
This deploy file is used to generate the matching page on micropython.org/download where the links will also be generated. This file should only give instructions, not links.

Reply:
Why are you people necroposting on this pull. Just delete this pull request.